### ISSUE TITLE
[Strand of the Ancients] Player Falls Through Boat

### ISSUE THREAD

**Issue by MaelstromAbbey** [gamefreedomgit/Maelstrom#3119]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
Encountered an unexpected issue where after joining the battleground Strand of the Ancients, character properly loaded onto the boat headed to the green gate, then unexpectedly the boat disappeared and character fell into the water. Other players did not appear to be affected. 

**How to reproduce:**
Queued for random battleground and loaded into Strand of the Ancients. This was the 1st round of the battleground where we were attacking. Character loaded onto the boat and could see other players when the boat abruptly disappeared. Happened on the boat headed toward the Green gate.  

**How it should work:**
Character should never fall through the boats headed toward the beach in Strand of the Ancients.

**Database links:**
https://cata-twinhead.twinstar.cz/?zone=4384#quests

**Comment by UltraNix** [comment]
@MaelstromAbbey After ships are started, you should be teleported back to ship. Was that the case?

**Comment by MaelstromAbbey** [comment]
@UltraNix, no, there was no teleport. I joined right as the ship began moving, though. In my case I loaded onto the ship, cast a party buff, then straight plummeted into the water. The boat disappeared, I didn't see it actually begin moving.

**Comment by UltraNix** [comment]
Fixed.
